runtime/onert/backend/gpu_cl/ClTensorManager.h

```diff
@@ -201,13 +199,14 @@ ClTensorManager<T_ITensor, T_Tensor>::atR(const ir::OperandIndex &ind)
 }
 
 template <typename T_ITensor, typename T_Tensor>
-InferenceContextEx::TensorReserver &ClTensorManager<T_ITensor, T_Tensor>::constTensorReservers(void)
+InferenceContextEx::TensorReserverEx &
+ClTensorManager<T_ITensor, T_Tensor>::constTensorReservers(void)
 {
   return _const_mgr->tensorReservers();
 }
```

Comment:
Fix return type: `_const_mgr->tensorReservers()` return `TensorReserverEx` type, not `TensorReserver`.